nomad/utils/text.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import re
+import unicodedata
+
+
+_punct_re = re.compile(r'[\t !"#$%&\'()*\-/<=>?@\[\\\]^_`{|},.]+')
+
+
+def slugify(text):
+    """ Returns a ASCII-only slug version of the considered text. """
+    result = []
+    for word in _punct_re.split(text.lower()):
+        word = unicodedata.normalize('NFKD', word).encode('ascii', 'ignore').decode('ascii')
```

Comment:
We are still using project names to generate our LXD hostnames (because it's a good practice to prefix our container names with the name of the related project). If the project name is set using the homedir name, this values must be "slugified". The question is: should we enforce the presence of the project's name in the config file (in that case we would assume that the related value is already a slug, so the slugify tool would be useless)?